docs/gallery/domain/plot_behavior.py

```diff
@@ -15,7 +15,7 @@
 * create an object:
 
   * :py:class:`~pynwb.base.TimeSeries` for continuous time series data,
-  * :py:class:`~pynwb.base.SpatialSeries` for continuous spatial data (e.g. position, direction relative to some reference frame),
+  * :py:class:`~pynwb.behavior.SpatialSeries` for continuous spatial data (e.g. position, direction relative to some reference frame),
```

Comment:
We noticed some incorrect class references with @bendichter in the behavior tutorial. I thought they would fit in this PR, let me know if not.

Reply:
@weiglszonja sure, go ahead, feel free to add changes to this PR as you see fit. I noticed a few link issues while working on another PR, and figured I'd be a good citizen and make a PR to fix what I found.

Reply:
thank you!

Reply:
@oruebel, it is not related here, but the behavior tutorial lacks a thumbnail image. do you create these thumbnails usually for the tutorials?

Reply:
I opened https://github.com/NeurodataWithoutBorders/pynwb/pull/1473 to make the thumbnail. I use the PowerPoint file at ``pynwb/docs/source/figures/gallery_thumbnails.pptx`` to create the thumbnails and then export as PNG. This may not be the optimal workflow, but its simple and keeps all thumbnails in one place where we can edit them.